lib/node_modules/@stdlib/blas/base/srotg/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@stdlib/blas/base/srotg",
+  "version": "0.0.0",
+  "description": "Construct a Givens plane rotation.",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "The Stdlib Authors",
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "browser": "./lib/main.js",
```

Comment:
```suggestion
```

Currently, this would break browser bundles, as the `assign` method would be excluded from the main export. This is found in other BLAS packages as `index.js` chooses between a JS vs native implementation. In the current implementation, we don't have that.